citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -532,7 +536,9 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
       			out.println("\t\t\t</table>");
        		    out.println("\t\t</div>");
 
-			    out.println("\t\t<div class=\"availabilityHeader\"></div>");
+
+			    out.println("\t\t<div class=\"availabilityHeader\" style=\"color:" +
+					    ServerConfigurationService.getString("official.institution.background.colour") + "\">" + "<strong>Availability</strong>" + "</div>");
```

Comment:
I'll make another Jira to prefix the property name with 'citations.' or the appropriate prefix